app/views/galleries/index.rb

```diff
@@ -91,7 +98,19 @@ def published_gallery_topic_options
     end
 
     def galleries_content
-      @galleries.map { |gallery| gallery_content(gallery) }
+      res = @galleries.map { |gallery| gallery_content(gallery) }
+      if res.length > 2
```

Comment:
Galleries do not necessarily have a description, so this "clicktip" may be pointing to... nothing, i.e. no _i_ symbol on the second gallery if it has no description. You need to check if the second has a description.